src/components/Select/styles.scss

```diff
@@ -159,8 +171,13 @@
       }
 
       &:active {
-        border: $tw-sz-frm-fld-text-input-stroke solid $tw-clr-brdr-warning;
-        outline: $tw-sz-frm-fld-text-input-stroke solid $tw-clr-brdr-warning;
+        border: math.div($tw-sz-frm-fld-text-input-stroke-active, 2) solid $tw-clr-brdr-warning;
+        outline: math.div($tw-sz-frm-fld-text-input-stroke-active, 2) solid $tw-clr-brdr-warning;
+      }
+
+      &:focus-within {
```

Comment:
Are you using focus-within instead of focus for any particular reason?

Reply:
The focus is applied to the input within the text field.  Need to apply the styles to the textfield.

Reply:
Excellent!